LibraryApp/src/components/navigators.js

```diff
@@ -32,6 +33,7 @@ const Navigators = () => {
         />
         <Stack.Screen name={libraryForm} component={Form} />
         <Stack.Screen name={showBookDetails} component={ShowBook} />
+        <Stack.Screen name={scanQRCode} component={ScanCode} options={{title: 'SCAN QR CODE'}}/>
```

Comment:
can we open this in modal? using react-navigation only

Reply:
Done